### PR TITLE
Update Firmware_ro.po for MK3_3.12

### PR DESCRIPTION
**Pull request by 3d-gussner**
Cherry-pick from https://github.com/prusa3d/Prusa-Firmware/pull/3632 and fixed few issues (removed mmu2) related changes.

- [X] Verified with `./lang-check.py --map firmware_MK3S.map po/Firmware_ro.po`
- 
All credits are going to @Hauzman and @leptun as they translated and reviewed the messages! :bow:

:2nd_place_medal:  As they are the 2n community translators updated the messages for FW 3.12.0 and upcoming version

### CODE REVIEW

File: lang/po/Firmware_ro.po [hunk 1]
```diff
@@ -81,13 +81,13 @@ msgstr "Asist."
 #. MSG_AUTO c=6
 #: ../../Firmware/messages.cpp:157 ../../Firmware/ultralcd.cpp:5864
 msgid "Auto"
-msgstr ""
+msgstr "Auto"
```
Comment (leptun) [inline lang/po/Firmware_ro.po:84]: @3d-gussner can this be translated in another way to indicate an identical translation? If I recall correctly, we could \0 the string to indicate that.

File: lang/po/Firmware_ro.po [hunk 2]
```diff
@@ -259,7 +259,7 @@ msgstr "Card scos"
 #. MSG_CNG_SDCARD c=18
 #: ../../Firmware/ultralcd.cpp:5538
 msgid "Change SD card"
-msgstr ""
+msgstr "Schimba card SD"
```
Comment (leptun) [inline lang/po/Firmware_ro.po:262]: Do we acutually use this translation? I thought it is ifdeffed out using the configuration.

File: lang/po/Firmware_ro.po [hunk 3]
```diff
@@ -435,7 +435,7 @@ msgstr ""
 #. MSG_EXTRUDER_CORRECTION c=13
 #: ../../Firmware/ultralcd.cpp:4214
 msgid "E-correct:"
-msgstr ""
+msgstr "E-correct:"
```
Comment (leptun) [inline lang/po/Firmware_ro.po:438]: Maybe "E-corectie:"?

File: lang/po/Firmware_ro.po [hunk 4]
```diff
@@ -1960,7 +1960,7 @@ msgstr "Corect. X:"
 #. MSG_XFLASH c=18
 #: ../../Firmware/ultralcd.cpp:5596
 msgid "XFLASH init"
-msgstr ""
+msgstr "Init XFLASH"
```
Comment (leptun) [inline lang/po/Firmware_ro.po:1963]: This message looks like debug. Should it be translated?

File: lang/po/Firmware_ro.po [hunk 5]
```diff
@@ -1988,7 +1988,8 @@ msgstr ""
 #: ../../Firmware/ultralcd.cpp:3317
 msgid ""
 "XYZ calibration compromised. Left front calibration point not reachable."
-msgstr ""
+msgstr "Calibrarea XYZ compromisa. Punctele de calibrare din fata stanga nu pot fi"
+"atinse."
```
Comment (leptun) [inline lang/po/Firmware_ro.po:1992]: Why is this multi-line? Is this ok?

File: lang/po/Firmware_ro.po [hunk 6]
```diff
@@ -1867,7 +1867,7 @@ msgstr "Optiuni"
 #: ../../Firmware/messages.cpp:111 ../../Firmware/ultralcd.cpp:5564
 #: ../../Firmware/ultralcd.cpp:5578
 msgid "Unload filament"
-msgstr "Descarca filament"
+msgstr "Descarca filam."
```
Comment (leptun) [inline lang/po/Firmware_ro.po:1870]: I don't remember, why is this shorter? Seems to be within limits

File: lang/po/Firmware_ro.po (hunk 2 above)
Comment (3d-gussner) [reply lang/po/Firmware_ro.po:262]: ifdeffed out  strings should be changed from _i  to _n to make it easier for the lang scripts.

File: lang/po/Firmware_ro.po (hunk 1 above)
Comment (3d-gussner) [reply lang/po/Firmware_ro.po:84]: If the translation is same as origin then the lang scripts "skip" it

File: lang/po/Firmware_ro.po (hunk 3 above)
Comment (3d-gussner) [reply lang/po/Firmware_ro.po:438]: > Maybe "E-corectie:"?

@Hauzman you decide

File: lang/po/Firmware_ro.po (hunk 4 above)
Comment (3d-gussner) [reply lang/po/Firmware_ro.po:1963]: Then make it `_n`

File: lang/po/Firmware_ro.po (hunk 3 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po:438]: I will changed it to E corectie
I Already started yesterday to re check the ro language after i started the windows build 😅 it is more easy now and some practice for me

File: lang/po/Firmware_ro.po (hunk 6 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po:1870]: I will check it I have seen also yesterday when playing with the windows build tha some translation difere from what i had from last modif.

Comment (Hauzman) [reply lang/po/Firmware_ro.po:1870]: I have checked and and I have change it  it gives error  that is wy i chnage it 
[E]: Text is longer than definition on line 2232: cols=16 rows=1 (rows diff=1)
 source text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅
   |Unload filament |
 translated text:
    ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅
   |Descarca filamen|t

File: lang/po/Firmware_ro.po (hunk 1 above)
Comment (leptun) [reply lang/po/Firmware_ro.po:84]: Good to know. Smart scripts :)

File: lang/po/Firmware_ro.po (hunk 2 above)
Comment (leptun) [reply lang/po/Firmware_ro.po:262]: Please make a note about this somewhere so we do this for all languages

File: lang/po/Firmware_ro.po (hunk 4 above)
Comment (leptun) [reply lang/po/Firmware_ro.po:1963]: Same as for other text

File: lang/po/Firmware_ro.po (hunk 6 above)
Comment (leptun) [reply lang/po/Firmware_ro.po:1870]: Understandable.

**Review by leptun — APPROVED** [review]
Approved, waiting for the final correction from @Hauzman before merging

**Comment by Hauzman** [comment]
@3d-gussner where should i push the final changes  to MK3 rep or the MK3_3.12

**Comment by 3d-gussner** [comment]
Good morning to the MK3 as this is the base.
I will then cherry-pick the commit to this pull request.

**Comment by Hauzman** [comment]
I have updated and push to my repo mk3D_Romanian will make a pull request after @3d-gussner  will check it as we discus it yesterday if all is ok then will make the new  pull request

**Review by DRracer — APPROVED** [review]

**Pull request merged**